apps/emqx/src/emqx_schema.erl

```diff
@@ -939,9 +939,9 @@ ssl(Defaults) ->
        sc(boolean(),
           #{ default => Df("secure_renegotiate", true)
            , desc => """
-SSL parameter renegotiation is a feature that allows a client and a server 
-to renegotiate the parameters of the SSL connection on the fly. 
-RFC 5746 defines a more secure way of doing this. By enabling secure renegotiation, 
+SSL parameter renegotiation is a feature that allows a client and a server
```

Comment:
I'm not quite sure if the space at the end can be removed. I think it might be used to format the paragraph.